hacker/admin.py

```diff
@@ -21,47 +46,83 @@ class Meta:
             'status':forms.RadioSelect,
         }
 
-
 class ApplicationAdmin(admin.ModelAdmin):
     form = ApplicationAdminForm
+    list_display = ('get_last_name','get_first_name','major', 'classification', 'grad_year', 'approved', 'get_is_active')
     fieldsets = [
         ('Related Objects',         {'fields': ['hacker']}),
         ('Personal Information',    {'fields': ['gender','major','classification','grad_year']}),
         ('Free Response Questions', {'fields': ['interests','essay','notes']}),
         ('Status',                  {'fields': ['approved']}),
     ]
 
+    make_approved.short_description = "Approve Selected Applications"
+    make_rejected.short_description = "Reject Selected Applications"
+    actions = [make_approved, make_rejected]
 
+    def has_add_permission(self, request, obj=None):
+        return True
+
+    def has_change_permission(self, request, obj=None):
+        return False
+
+#    def has_delete_permission(self, request, obj=None):
+#        return False
+
+
+''' `Confirmation` '''    
```

Comment:
This comment doesn't really provide any information that people can't gather from the code, I'd remove this.